src/azure-cli-testsdk/azure/cli/testsdk/base.py

```diff
@@ -124,6 +126,31 @@ def cmd(self, command, checks=None):
 
         return result
 
+    def create_temp_file(self, size_kb):
```

Comment:
Can we make the size optional so anyone can just request a temp file and know it will get deleted after the test?